src/game/level/boxes.c

```diff
@@ -8,6 +8,9 @@
 #include "system/lt.h"
 #include "system/nth_alloc.h"
 #include "system/log.h"
+#ifdef WINDOWS_OS
+#include <stdio.h> // for EOF macro
```

Comment:
`stdio.h` exists on all of the platforms. Just include it without `#ifdef WINDOWS_OS`